### ISSUE TITLE
TokenCompleteTextView - tokens not accessible in Talkback

### ISSUE THREAD

**Issue by sagar-waghmare-rsl** [splitwise/TokenAutoComplete#265]
TokenCompleteTextView - tokens are not accessible in Talkback. It says ",," [comma comma] when it should say the selected tokens. Also, individual tokens are not focused (for accessibility).

You may want to use ExploreByTouchHelper for adding the support.
![device-2016-11-29-203150](https://cloud.githubusercontent.com/assets/12628247/20725148/c46ed114-b625-11e6-8783-90df022e65a8.png)

**Comment by lynamemi** [comment]
+1 for this. We're trying to add our own ExploreByTouch helper, but without access to the private view in `TokenImageSpan` this is proving to be quite challenging.

**Comment by mgod** [comment]
I'm not sure the view in the `TokenImageSpan` will continue to exist in future versions (it may be replaced with a `Drawable`), so I'd like to explore options that are not dependent on that.
